alephium/contracts/token_bridge/token_bridge_governance.ral

```diff
@@ -115,4 +116,20 @@ Abstract Contract TokenBridgeGovernance(
         let consistencyLevel = u256From1Byte!(byteVecSlice!(payload, 33, 34))
         minimalConsistencyLevel = consistencyLevel
     }
+
+    @using(readonly = true, externalCallCheck = false)
+    pub fn getRefundAddress() -> Address {
+        return refundAddress
+    }
+
+    pub fn updateRefundAddress(vaa: ByteVec) -> () {
+        let (targetChainId, payload) = parseAndVerifyGovernanceVAA(vaa, ActionId.UpdateRefundAddress)
+        assert!(targetChainId == localChainId, ErrorCodes.InvalidTargetChainId)
+        let addressBytes = byteVecSlice!(payload, 33, 66)
```

Comment:
Let's maybe support arbitrary address (addressBytesLength ++ addressBytes). I will probably use a multi-sig address for this.

Reply:
Done!